lib/quantity/math.ex

```diff
@@ -222,4 +222,56 @@ defmodule Quantity.Math do
   def round(quantity, decimal_count) do
     Quantity.new(Decimal.round(quantity.value, decimal_count, :half_up), quantity.unit)
   end
+
+  defp reduce_unit(unit) do
+    {numerators, denominators} =
+      unit
+      |> isolate_units({[], []})
+      |> shorten()
+
+    case {numerators, denominators} do
+      {[], []} -> 1
+      {[a], []} -> a
+      {[], [a]} -> {:div, 1, a}
+      {[a, b], []} -> {:mult, a, b}
+      {[a], [b]} -> {:div, a, b}
+      # Everything below here is not valid, but we try anyway
+      {as, []} -> Enum.reduce(as, &{:mult, &1, &2})
+      {[], bs} -> {:div, 1, Enum.reduce(bs, &{:mult, &1, &2})}
+      {as, bs} -> {:div, Enum.reduce(as, &{:mult, &1, &2}), Enum.reduce(bs, &{:mult, &1, &2})}
+    end
+  end
+
+  defp shorten({numerators, denominators}) do
```

Comment:
1) Suggestion: Implement

```
defp shorten(numerators, []), do: {numerators, []}
defp shorten([], denominators), do: {[], denominators}
```

and implement the sorting as a separate function


2. Suggested comment for `shorten/2`:
```
# Remove common elements in numerator and denominator
```

Reply:
Changes in bc1785f

1. Agree that sorting should be it's own thing :+1: I don't agree to add extra function clauses for cases that are already covered in the main case. What is the reason for that?
2. :+1: